subt_ign/src/GameLogicPlugin.cc

```diff
@@ -640,6 +644,31 @@ void GameLogicPlugin::PostUpdate(
     std::map<std::string, ignition::math::Pose3d>::iterator baseIter =
       this->dataPtr->poses.find(subt::kBaseStationName);
 
+    _ecm.Each<gazebo::components::DetachableJoint>(
```

Comment:
We use detachable joints for making breadcrumbs and dynamic obstacles static after a period of time. We can maybe filter them out by checking if both the parent and child models are also performers.

Reply:
I think this fixes it: 9d1b038f198dab3f825d7780e6503e657076ab51